src/nvim/memline.c

```diff
@@ -134,18 +136,22 @@ struct data_block {
 #define INDEX_SIZE  (sizeof(unsigned))      // size of one db_index entry
 #define HEADER_SIZE (sizeof(DATA_BL) - INDEX_SIZE)  // size of data block header
 
-#define B0_FNAME_SIZE_ORG       900     // what it was in older versions
-#define B0_FNAME_SIZE_NOCRYPT   898     // 2 bytes used for other things
-#define B0_FNAME_SIZE_CRYPT     890     // 10 bytes used for other things
-#define B0_UNAME_SIZE           40
-#define B0_HNAME_SIZE           40
+enum {
+  B0_FNAME_SIZE_ORG = 900,      // what it was in older versions
+  B0_FNAME_SIZE_NOCRYPT = 898,  // 2 bytes used for other things
+  B0_FNAME_SIZE_CRYPT = 890,    // 10 bytes used for other things
+  B0_UNAME_SIZE = 40,
+  B0_HNAME_SIZE = 40,
+};
 // Restrict the numbers to 32 bits, otherwise most compilers will complain.
 // This won't detect a 64 bit machine that only swaps a byte in the top 32
 // bits, but that is crazy anyway.
-#define B0_MAGIC_LONG   0x30313233L
-#define B0_MAGIC_INT    0x20212223L
-#define B0_MAGIC_SHORT  0x10111213L
-#define B0_MAGIC_CHAR   0x55
+enum {
+  B0_MAGIC_LONG = 0x30313233L,
+  B0_MAGIC_INT = 0x20212223L,
+  B0_MAGIC_SHORT = 0x10111213L,
+  B0_MAGIC_CHAR = 0x55,
+};
```

Comment:
Is this OK? Since enums are supposed to be ints or something, does this cause a problem?